### PR TITLE
refactor: Update `droplet` icon and add `flame` icon

### PR DESCRIPTION
**Pull request by ericfennis**
# Droplet icon and new flame icon
closes [#449](https://github.com/feathericons/feather/issues/449)
image
100% 
image

## Design
The Flame and droplet uses the same "body", so it match.
image

## Original vs New droplet icon
image

### CODE REVIEW

**Comment by lscheibel** [comment]
This one could be merged right? I like it

File: icons/flame.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M8.49997 14.5C9.88068 14.5 11 13.3807 11 12C11 10.6193 10.5 10 9.99997 9C8.9283 6.85666 9.77608 4.9459 12 3C12.5 5.5 14 7.9 16 9.5C18 11.1 19 13 19 15C19 18.866 15.866 22 12 22C8.13401 22 5 18.866 5 15C5 13.847 5.43297 12.706 5.99997 12C5.99997 13.3807 7.11926 14.5 8.49997 14.5Z" />
```
Comment (jletey) [inline icons/flame.svg:12]: Minor nitpick here @ericfennis 😅 Indentation of 2 not 4

```suggestion
  <path d="M8.49997 14.5C9.88068 14.5 11 13.3807 11 12C11 10.6193 10.5 10 9.99997 9C8.9283 6.85666 9.77608 4.9459 12 3C12.5 5.5 14 7.9 16 9.5C18 11.1 19 13 19 15C19 18.866 15.866 22 12 22C8.13401 22 5 18.866 5 15C5 13.847 5.43297 12.706 5.99997 12C5.99997 13.3807 7.11926 14.5 8.49997 14.5Z" />
```

Comment (ericfennis) [reply icons/flame.svg:12]: Sorry my fault, I know it's not the first time 😅 . I think we need an `.editorconfig`. 
My tabs are constantly switching spaces.

File: icons/droplet.svg [hunk 2]
```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M12 2.69l5.66 5.66a8 8 0 1 1-11.31 0z" />
+  <path d="M12 22C15.866 22 19 18.866 19 15C19 13 18 11.1 16 9.5C14 7.9 12.5 5.5 12 3C11.5 5.5 10 7.9 8 9.5C6 11.1 5 13 5 15C5 18.866 8.13404 22 12 22Z" />
```
Comment (jletey) [inline icons/droplet.svg:12]:
```suggestion
  <path d="M12 22a7 7 0 0 0 7-7c0-2-1-3.9-3-5.5s-3.5-4-4-6.5c-.5 2.5-2 4.9-4 6.5C6 11.1 5 13 5 15a7 7 0 0 0 7 7z" />
```

File: icons/flame.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M8.49997 14.5C9.88068 14.5 11 13.3807 11 12C11 10.6193 10.5 10 9.99997 9C8.9283 6.85666 9.77608 4.9459 12 3C12.5 5.5 14 7.9 16 9.5C18 11.1 19 13 19 15C19 18.866 15.866 22 12 22C8.13401 22 5 18.866 5 15C5 13.847 5.43297 12.706 5.99997 12C5.99997 13.3807 7.11926 14.5 8.49997 14.5Z" />
```
Comment (jletey) [inline icons/flame.svg:12]:
```suggestion
  <path d="M8.5 14.5A2.5 2.5 0 0 0 11 12c0-1.38-.5-2-1-3-1.072-2.143-.224-4.054 2-6 .5 2.5 2 4.9 4 6.5 2 1.6 3 3.5 3 5.5a7 7 0 1 1-14 0c0-1.153.433-2.294 1-3a2.5 2.5 0 0 0 2.5 2.5z" />
```

**Comment by ericfennis** [comment]
@jletey Thanks for the optimizations!

**Comment by jletey** [comment]
@ericfennis Thanks for the stunning icons 🔥

**Pull request merged**